build/tasks/build.js

```diff
@@ -82,13 +89,15 @@ module.exports = function( grunt ) {
 			contents = contents
 				.replace( /define\(\[[^\]]*\]\)[\W\n]+$/, "" );
 		}
+
 		// AMD Name
-		if ( (amdName = grunt.option( "amd" )) != null && /^exports\/amd$/.test( name ) ) {
-			if (amdName) {
+		if ( ( amdName = grunt.option( "amd" ) ) != null && /^exports\/amd$/.test( name ) ) {
```

Comment:
I wasn't advocating blocking the PR, just backing out the changes pertaining to spaces in parenthesized expressions (which I believe would shrink it considerably, but I suppose would require a temporary `requireSpacesInsideParentheses` override).

As for revoting/updating documentation/etc., a rule requiring such spaces doesn't appear in http://contribute.jquery.org/style-guide/js/#spacing or the old [.jscs.json](https://github.com/markelog/jquery/commit/b7283d738ec4c2d9d87cb16c32a3209cab567ed1#diff-ed6a67a86c4c5aa1dd5fcb0acb6f5c67), and in fact was stealth-added (from the perspective of this codebase, not its contributors) by https://github.com/jscs-dev/node-jscs/commit/0f78baf08fba75aeba4575608aaa08c384875701 in response to https://github.com/jquery/contribute.jquery.org/pull/104 (which was about requiring spaces in function calls). Since you committed it, I can just directly ask how you would have treated _expression_ parentheses if JSCS were capable of differentiating them. But I will point out that I haven't encountered one argument in favor of requiring them that didn't hinge upon style guide automation, which falls apart as soon as that process becomes sufficiently sophisticated.